packages/frontend/src/features/sqlRunner/transformers/usePieChart.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    PieChartDataTransformer,
+    type PieChartSQLConfig,
+    type ResultRow,
+    type SqlColumn,
+} from '@lightdash/common';
+import { useMemo } from 'react';
+import { useAsync } from 'react-use';
+import { SqlRunnerResultsTransformerFE } from './SqlRunnerResultsTransformerFE';
+
+export const usePieChart = (
```

Comment:
This is basically the same as useBarChart. We could probably be fine with one useSQLChart

Reply:
agree, got a similar comment here: https://github.com/lightdash/lightdash/pull/10915/files#r1698085158

Reply:
Cool, added a ticket I will do next: https://github.com/lightdash/lightdash/issues/10926